mkfifo/src/main.rs

```diff
@@ -0,0 +1,19 @@
+use clap::{load_yaml, App, AppSettings::ColoredHelp};
+use coreutils_core::mkfifo::mkfifo;
+use std::process;
+
+fn main() {
+    let yaml = load_yaml!("mkfifo.yml");
+    let matches = App::from_yaml(yaml).settings(&[ColoredHelp]).get_matches();
+    let filepath = matches.value_of("NAME").expect("No filename provided.");
+    let mode = matches.value_of("mode").unwrap();
+    let mode: u32 = u32::from_str_radix(mode, 8)
+        .unwrap_or_else(|_| panic!("Mode '{}' is not an octal number.", mode));
```

Comment:
Panic will make the program exit with code 101, the mkfifo tool returns exit code 1 when mode is not a octal number

Reply:
You're completely right, my last commit should fix the wrong behavior. Thanks for checking it out :)